dotnet/xml/ns-Microsoft.Identity.Client.Broker.xml

```diff
@@ -1,6 +1,72 @@
 <Namespace Name="Microsoft.Identity.Client.Broker">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary overwrite="false">
+        <format type="text/markdown"><![CDATA[
+[![NuGet](https://img.shields.io/nuget/v/Microsoft.Identity.Client.Broker.svg?style=flat-square&label=Microsoft.Identity.Client.Broker&colorB=00b200)](https://www.nuget.org/packages/Microsoft.Identity.Client.Broker/)  [![NuGet downloads](https://img.shields.io/nuget/dt/Microsoft.Identity.Client.Broker?style=flat-square)](https://www.nuget.org/packages/Microsoft.Identity.Client.Broker/) [![GitHub Repo stars](https://img.shields.io/github/stars/AzureAD/microsoft-authentication-library-for-dotnet?style=flat-square)](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet)
+
+The `Microsoft.Identity.Client.Broker` is the namespace in the Microsoft Authentication Library (MSAL) for .NET that handles interactions with authentication brokers. It is available through the [`Microsoft.Identity.Client.Broker`](https://www.nuget.org/packages/Microsoft.Identity.Client.Broker/) package.
```

Comment:
idk if we need to clarify, since like using mobile brokers is included in the main package.

```suggestion
The `Microsoft.Identity.Client.Broker` is the namespace in the Microsoft Authentication Library (MSAL) for .NET that handles interactions with desktop authentication brokers. It is available through the [`Microsoft.Identity.Client.Broker`](https://www.nuget.org/packages/Microsoft.Identity.Client.Broker/) package.
```